.circleci/config.yml

```diff
@@ -50,43 +45,46 @@ jobs:
           path: ~/reports
   yamllint:
     docker:
-      - image: cimg/python:3.9.2
+      - image: cimg/python:3.9.6
     steps:
       - checkout
       - run: pip install yamllint
       - run: yamllint -d .yamllint.yml .
   shellcheck:
     docker:
-      - image: cimg/base:2021.03
+      - image: cimg/base:2021.07
     steps:
       - checkout
       - shellcheck/install
       - shellcheck/check
   audit:
     docker:
-      - image: cimg/node:14.16.0
+      - image: cimg/node:14.17.3
     steps:
       - checkout
       - run: npm audit --production
-      - slack/status:
-          fail_only: true
-          failure_message: The audit check for \`$CIRCLE_PROJECT_REPONAME\` has failed.
+      - slack/notify:
+          event: fail
+          template: basic_fail_1
   docker-build-and-push:
     docker:
-      - image: cimg/node:14.16.0
+      - image: cimg/node:14.17.3
     steps:
       - checkout
       - setup_remote_docker:
-          version: 20.10.2
+          version: 20.10.6
       - node/install-packages
       - run: npx semantic-release
       - run: docker build  --tag aegee/discounts:$(node -p "require('./package.json').version") --tag aegee/discounts:latest -f docker/discounts/Dockerfile .
       - run: docker login --username $DOCKER_LOGIN --password $DOCKER_PASSWORD
       - run: docker push aegee/discounts:$(node -p "require('./package.json').version")
       - run: docker push aegee/discounts:latest
-      - slack/status:
-          success_message: The Docker image for \`$CIRCLE_PROJECT_REPONAME\` with tags \`$(node -p "require('./package.json').version")\` and \`latest\` has been pushed to Dockerhub.
-          failure_message: The Docker image for \`$CIRCLE_PROJECT_REPONAME\` build and push has failed.
+      - slack/notify:
+          event: pass
+          template: basic_success_1
```

Comment:
what will this output?

Reply:
Something like this;
![](https://github.com/CircleCI-Public/slack-orb/wiki/img/templates/basic_success_1.png)

So basically it will tell us that docker-push-and-build passed for the master branch of discounts

Reply:
can it be customized? it'd be nice for us to see which version was built and pushed, like in the setup I did

Reply:
We can make custom notifications with the [Block Kit Builder](https://app.slack.com/block-kit-builder) from Slack. I'll take a look to see if I can make something similar to what we have now

Reply:
Okay, made a custom notification. Not sure how I could get the green bar in front that the old message had, but this should also suffice imo. Name and avatar of the app can be changed in Slack, no code needed for that
![slack_2021-07-20_00-40-27](https://user-images.githubusercontent.com/13023439/126236488-2cdd491d-f83b-4529-94b9-99df625b1278.png)

Reply:
@serge1peshcoff is the message above good enough or do you want a visual indicator as well to see if it was successfull?

Reply:
lgtm